src/components/ContentLayout/ContentContainer/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native';
+import { device, appHeaderHeight } from '../../../constants';
+
+const constainerMargin = 10;
+
+export default StyleSheet.create({
+  container: {
+    margin: constainerMargin,
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: device.windowHeight - device.statusBarHeight - appHeaderHeight - 2 * constainerMargin
```

Comment:
The `statusBarHeight` here is redundant I believe, because for Android the window dimension will exclude the size used by the status bar (if not translucent) and bottom navigation bar (https://reactnative.dev/docs/dimensions#displaymetrics)